inspire_dojson/journals/rules.py

```diff
@@ -27,88 +27,209 @@
 from dojson import utils
 from idutils import normalize_issn
 
-from inspire_utils.helpers import maybe_int
+from inspire_utils.helpers import force_list, maybe_int
 
 from .model import journals
 from ..utils import get_record_ref
 
 
-@journals.over('issn', '^022..')
+@journals.over('issns', '^022..')
 @utils.for_each_value
-def issn(self, key, value):
-    try:
-        issn = normalize_issn(value['a'])
-    except KeyError:
-        return {}
-
-    b = value.get('b', '').lower()
-    if 'online' == b:
-        medium = 'online'
-        comment = ''
-    elif 'print' == b:
-        medium = 'print'
-        comment = ''
-    elif 'electronic' in b:
-        medium = 'online'
-        comment = 'electronic'
-    elif 'ebook' in b:
-        medium = 'online'
-        comment = 'ebook'
-    elif 'hardcover' in b:
-        medium = 'print'
-        comment = 'hardcover'
-    else:
-        medium = ''
-        comment = b
+def issns(self, key, value):
+    def _get_issn(value):
+        return normalize_issn(value.get('a'))
+
+    def _get_medium(value):
+        MEDIUM_MAP = {
+            'Online (English)': 'online',
+            'Online (Russian': 'online',
+            'Online': 'online',
+            'Print (Russian)': 'print',
+            'Print': 'print',
+            'e-only': 'online',
+            'electronic': 'online',
+            'online': 'online',
+            'print': 'print',
+            'printed': 'print',
+        }
+
+        return MEDIUM_MAP.get(value.get('b'))
+
+    issn = _get_issn(value)
+    medium = _get_medium(value)
 
+    if issn:
+        return {
+            'medium': medium,
+            'value': issn,
+        }
+
+
+@journals.over('journal_title', '^130..')
+def journal_title(self, key, value):
     return {
-        'comment': comment,
-        'medium': medium,
-        'value': issn,
+        'title': value.get('a'),
+        'subtitle': value.get('b'),
     }
 
 
-@journals.over('coden', '^030..')
+@journals.over('related_records', '^530..')
 @utils.for_each_value
-def coden(self, key, value):
-    return value.get('a')
+def related_records(self, key, value):
+    def _get_relation(value):
+        RELATION_MAP = {
+            'a': 'predecessor',
+            'b': 'other',
```

Comment:
this should be changed once https://github.com/inspirehep/inspire-next/issues/2433 is fixed.